ebpfsvc/rpc_api.cpp

```diff
@@ -36,8 +40,19 @@ ebpf_server_verify_and_load_program(
         info->handle_map,
         info->instruction_count,
         reinterpret_cast<ebpf_inst*>(info->instructions),
-        const_cast<const char**>(logs),
-        logs_size);
+        &ebpf_logs,
+        &ebpf_logs_size);
+
+    if (ebpf_logs) {
+        // The ebpf_logs buffer was allocated by the ebpf allocator whereas we
+        // must return a string allocated by the MIDL allocator.
+        *logs = (char*)MIDL_user_allocate(ebpf_logs_size);
```

Comment:
It looks like there are 2 versions of `MIDL_user_allocate()` currerntly in the repo. The client version is calling `ebpf_allocate()`, whereas the server version is calling `malloc()`.

If we change the implementation of the server version to also call `ebpf_allocate()`, then do we still need this change?

Reply:
Done